samples/msal-node-samples/ElectronTestApp/src/AuthProvider.ts

```diff
@@ -118,42 +116,60 @@ export default class AuthProvider {
         return authResponse.accessToken || null;
     }
 
-    async getTokenSilent(tokenRequest: SilentFlowRequest): Promise<AuthenticationResult> {
+    async getTokenSilent(
+        tokenRequest: SilentFlowRequest
+    ): Promise<AuthenticationResult> {
         try {
-            return await this.clientApplication.acquireTokenSilent(tokenRequest);
+            return await this.clientApplication.acquireTokenSilent(
+                tokenRequest
+            );
         } catch (error) {
-            console.log("Silent token acquisition failed, acquiring token using pop up");
-            const authCodeRequest = { ...this.authCodeUrlParams, ...tokenRequest };
+            console.error(
+                "Silent token acquisition failed, acquiring token using pop up"
```

Comment:
```suggestion
                "Silent token acquisition failed, acquiring token interactively"
```

Reply:
This is out of scope for this PR. We can address this in a follow-up PR.